### DISCUSSION TITLE
help me with that error

### DISCUSSION THREAD

**Discussion by samaremad** [serenity-is/Serenity#5561] [Help]
hello everyone 

i made button delete more than one records and it was worked fine but now i get this button with the same code and add it to another project but there strange error i dont know what it mean and how to avoid it 

![Screenshot (91)](https://user-images.githubusercontent.com/77009503/110235261-c2057b00-7f37-11eb-83ac-d387db3e6b96.png)

**Comment by minhhungit** [comment]
I guess it relates to this discusstion https://github.com/serenity-is/Serenity/discussions/5448

I also answered on the discusstion
